src/cljs/main/broadfcui/main.cljs

```diff
@@ -120,68 +120,106 @@
                   "http://status.firecloud.org/"]
                  " for more information."])}))
 
-(defn status-alert [title message link]
-  [:div {:style {:borderBottom "1px solid" :borderBottomColor (:line-default style/colors)
-                 :backgroundColor (:exception-state style/colors) :padding "1rem"}}
-   [:div {:style {:display "flex" :background (:exception-state style/colors) :color "#fff"
-                  :alignItems "center" :marginBottom "0.5rem"}}
-    [icons/ExceptionIcon {:size 18}]
-    [:span {:style {:marginLeft "0.5rem" :fontWeight 600
-                    :verticalAlign "middle"}}
-     (if title title "Service Alert")]]
-   [:div {:style {:color "#fff" :fontSize "90%"}}
-    (str message " ")
-     (when link [:a {:href (str link) :target "_blank" :style {:color "#fff"}} "Read more..."])]])
-
 (defn status-alert-interval [attempt]
   (cond
     (= attempt 0) (config/status-alerts-refresh)
     (> attempt (config/max-retry-attempts)) (config/status-alerts-refresh)
     :else (utils/get-exponential-backoff-interval attempt)))
 
-(react/defc StatusAlertContainer
+(react/defc Banner
+  {:get-initial-state
+   (fn []
+     {:showing-more? false})
+   :render
+   (fn [{:keys [props state]}]
+     [:div {}
+      (let [{:keys [background-color text-color title message link more-content]} props]
+        (utils/cljslog more-content)
+        [:div {:style {:borderBottom "1px solid" :borderBottomColor (:line-default style/colors) :color text-color
+                       :backgroundColor background-color :padding "1rem"}}
+         [:div {:style {:display "flex" :alignItems "center" :marginBottom "0.5rem"}}
+          [icons/ExceptionIcon {:size 18 :color text-color}]
+          [:span {:style {:marginLeft "0.5rem" :fontWeight "bold"
+                          :verticalAlign "middle"}}
+           (if title title "Service Alert")]]
+         [:div {:style {:color text-color :fontSize "90%"}}
+          (str message " ")
+          (if more-content
+            [:div {} [:a {:href  "javascript:;" :onClick #(swap! state assoc :showing-more? (not (:showing-more? @state)))
+                          :style {:color "#000"}} (if (:showing-more? @state) "Hide details..." "Show details...")]])
```

Comment:
Break this up into multiple lines, plz. For instance, the contents of the `a` should have their own line.